pandora-client-web/src/components/chatroomAdmin/chatroomAdmin.scss

```
@@ -63,11 +63,36 @@
 	gap: 0.5em;
 
 	.header {
-		display: grid;
-		grid-template-columns: auto 1fr 1fr;
-		grid-template-rows: auto;
-		column-gap: 0.6em;
+		@include flex(column);
+		gap: 0.5em;
 		max-width: 95vw;
+
+		.header-filter {
+			display: grid;
+			grid-template-columns: auto 1fr;
+			grid-template-rows: auto;
+			column-gap: 0.6em;
+			width: 100%;
+
+			span {
+				margin: auto 0;
+			}
+		}
+
+		.header-tags {
+			@include flex(row);
```

This needs to wrap for narrow screens.
```suggestion
			@include flex(row, wrap);
```